src/Execution/Executor.php

```diff
@@ -588,27 +536,27 @@ private function completeValue(
 
     /**
      * @param AbstractTypeInterface $returnType
-     * @param array                 $fieldNodes
+     * @param FieldNode[]           $fieldNodes
      * @param ResolveInfo           $info
-     * @param                       $path
-     * @param                       $result
+     * @param array                 $path
+     * @param mixed                 $result
      * @return array|PromiseInterface
      * @throws ExecutionException
      * @throws InvalidTypeException
-     * @throws \Digia\GraphQL\Error\InvariantException
+     * @throws InvariantException
      * @throws \Throwable
      */
-    private function completeAbstractValue(
+    protected function completeAbstractValue(
         AbstractTypeInterface $returnType,
         array $fieldNodes,
         ResolveInfo $info,
-        $path,
+        array $path,
         &$result
     ) {
         $runtimeType = $returnType->resolveType($result, $this->context->getContextValue(), $info);
 
         if (null === $runtimeType) {
-            //@TODO Show warning
+            // TODO: Display warning
```

Comment:
Why you change `@TODO` to `TODO:`?

Reply:
Consistency.

Reply:
`@TODO` is usually used inside docblocks.